config/operator/base/deployment/deployment.yaml

```diff
@@ -27,41 +27,61 @@ spec:
       securityContext:
         runAsNonRoot: true
       containers:
-      - command:
-        - /operator
-        image: controller:latest
-        imagePullPolicy: IfNotPresent
-        name: manager
-        env:
-        - name: CHART_PATH
-          value: /module-chart
-        - name: SERVERLESS_MANAGER_UID
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.uid
-        securityContext:
-          allowPrivilegeEscalation: false
-          capabilities:
-            drop:
-              - "ALL"
-        livenessProbe:
-          httpGet:
-            path: /healthz
-            port: 8081
-          initialDelaySeconds: 15
-          periodSeconds: 20
-        readinessProbe:
-          httpGet:
-            path: /readyz
-            port: 8081
-          initialDelaySeconds: 5
-          periodSeconds: 10
-        resources:
-          limits:
-            cpu: 1000m
-            memory: 512Mi
-          requests:
-            cpu: 10m
-            memory: 64Mi
+        - command:
+            - /operator
+          image: controller:latest
+          imagePullPolicy: IfNotPresent
+          name: manager
+          env:
+            - name: CHART_PATH
+              value: /module-chart
+            - name: SERVERLESS_MANAGER_UID
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.uid
+            - name: IMAGE_FUNCTION_CONTROLLER
+              value: europe-docker.pkg.dev/kyma-project/prod/function-controller:main
```

Comment:
having fixed image folder and tag here in the ENVs fixes the image versions to main .. also for installations created via`IMG=xxx make install-serverless-custom-operator`
This is how we install serverless in our integration tests
https://github.com/kyma-project/serverless/blob/main/.github/workflows/_integration-tests.yaml#L69

We should rethink this as with the proposed code we wouldn't be testing the changes in controller, runtime images etc

Reply:
the values for the envs should be part of kustomization that is deployed on PR integration tests